src/vstest.console/Resources/Resources.resx

```diff
@@ -276,11 +276,14 @@
   <data name="FileNotFound" xml:space="preserve">
     <value>'{0}' not found.</value>
   </data>
+  <data name="Framework35NotSupported" xml:space="preserve">
+    <value>Framework35 not supported. Use Framework40 or above to run tests in CLR 4.0 "compatibly mode".</value>
```

Comment:
Suggestion:
Framework35 is not supported. For projects targeting .Net Framework 3.5, please use Framework40 to run tests in CLR 4.0 "compatibility mode".